.github/workflows/build.yml

```diff
@@ -42,14 +42,34 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: "1.1.4"
+      - name: Configure poetry
+        shell: bash
+        run: python -m poetry config virtualenvs.in-project true
```

Comment:
Is the `python -m` necessary? We don't use it lower.

Reply:
probably not. thanks. I will take it out